### ISSUE TITLE
Issue with dashboard navigation after clicking external links online

### ISSUE THREAD

**Issue by olivroy** [quarto-dev/quarto-cli#9411]
### Bug description

Navigation after opening external dashboard (hosted on Quarto-pub) link doesn't work well.

````qmd
---
title: title
format: dashboard
execute: 
  echo: false
---

# Title

## Row 

<https://github.com/quarto-dev/quarto-cli/issues/9411>

See it there.

Not a quartopub issue 

## Row

# Another page

## Row 

<https://github.com/quarto-dev/quarto-cli/issues/9411>


## Another

```{r}
ggplot2::ggplot()
```

````

### Steps to reproduce

1. Go to https://grrrck.quarto.pub/mynorfolk-dash/#about

2. Click on a link
 (source code for example)

3. Navigate back in browser (Alt + left)

### Expected behavior

I would expect that going back to the page (With the back button)

### Actual behavior

Get a blank page.
![image](https://github.com/quarto-dev/quarto-cli/assets/52606734/7a0af736-fb65-4c26-980b-f7f5c47275fe)

### Your environment

Web. I tested with MS Edge and Chrome

But it happened also for one of my dashboard created with Quarto 1.4.553 on WIndows and published on Quarto-pub.

IF I refresh, the correct page shows correctly again

With a local dashboard, I can't reproduce it. Only if published to QuartoPub at the moment.

gallery of dashboards to test other oneshttps://quarto.org/docs/dashboards/examples/

but got the same from https://isabelizimm.me/vetiverdashboard/# and https://mine-cetinkaya-rundel.github.io/ld-dashboard/

I can reproduce locally with quarto preview in RStudio

### Quarto check output

```
Quarto 1.4.553
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.4.553
      Path: C:\~\AppData\Local\Programs\RStudio\resources\app\bin\quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: (external install)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Using: TinyTex
      Path: C:\~\AppData\Roaming\TinyTeX\bin\windows\
      Version: 2024

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........OK
      Version: 4.3.2
      Path: C:/PROGRA~1/R/R-43~1.2
      LibPaths:
        - C:/Users/~/AppData/Local/R/win-library/4.3
        - C:/Program Files/R/R-4.3.2/library
      knitr: 1.46
      rmarkdown: 2.26

[>] Checking Knitr engine render......OK
```

**Comment by cscheid** [comment]
This behavior is by design - the navigation changes the URL so that you can share a link to a specific state of the dashboard panes. Is that what you're referring to?

**Comment by olivroy** [comment]
[Enregistrement-20240418_154352.webm](https://github.com/quarto-dev/quarto-cli/assets/52606734/49c2dcc1-4957-4500-a09c-cdf11a689c76)

I mean this. Is this expected? 

https://quarto.org/docs/output-formats/html-basics.html#external-links

Even if my link is external i.e.

```
[link](https://github.com/quarto-dev/quarto-cli/issues/9411){.external target="_blank"}
```

there seems that there is a redirection to a white page before opening the link, which makes navigation back and forth (between outside link and dashboard) (Navigation within the dashboard is fine, as there is no need for the browser nav buttons)

**Comment by cscheid** [comment]
I see what you mean. The external link behavior is definitely not good.

**Comment by cscheid** [comment]
Hm. I can't repro with https://carlos-scheidegger.quarto.pub/issue-9411-b/, which is rendered using this source:

````
---
title: title
format: dashboard
execute: 
  echo: false
---

# Title

## Row 

<https://github.com/quarto-dev/quarto-cli/issues/9411>

See it there.

Not a quartopub issue 

## Row

# Another page

## Row 

<https://github.com/quarto-dev/quarto-cli/issues/9411>


## Another

```{r}
ggplot2::ggplot()
```
````

**Comment by olivroy** [comment]
You are right! Somehow the problem was that the quarto-dashboard.js didn't update properly on my auto-update action. Fetching my repo locally, I see this diff. Committing this will fix my issue. 

![Image](https://github.com/user-attachments/assets/12413aba-42ad-4ba3-afe1-0d85b644b7c9)

Thanks for fixing and for getting back to me!
